src/core/commandVersionUpgrades/upgradeV2ToV3/targetDescriptorV2.types.ts

```diff
@@ -68,7 +68,7 @@ export type SurroundingPairName =
   | SimpleSurroundingPairName
   | ComplexSurroundingPairName;
 
-export type SimpleScopeTypeType =
+export type SimpleScopeTypeTypeV2 =
```

Comment:
@AndreasArvidsson I believe this was an oversight in #982, and caused a type error when a new scope type was added